UI/resources/js/AddAdvertisement.js

```diff
@@ -0,0 +1,188 @@
+// eslint-disable-next-line func-names
+(function () {
+  const MESSAGE = document.getElementById('name');
```

Comment:
Please, use lowercase names for both const and let, uppercase is used for shared consts and literals